src/mono/mono/metadata/metadata.c

```diff
@@ -46,12 +46,17 @@ typedef struct {
 	MonoGenericContext context;
 } MonoInflatedMethodSignature;
 
+enum {
+	MONO_TYPE_EQ_FLAGS_SIG_ONLY = 1,
+	MONO_TYPE_EQ_FLAG_IGNORE_CMODS = 2
```

Comment:
nit: usually it's better to add a trailing `,` to the last enum value, in case we want to add more later. it'll make the subsequent diff a bit cleaner.

Reply:
Fixed